web/src/app/services/ServiceRules/ServiceRuleForm.tsx

```diff
@@ -300,6 +300,48 @@ export default function ServiceRuleForm(
     )
   }
 
+  const renderContents = (
+    actionIdx: number,
+    action: ServiceRuleActionInput,
+  ): React.ReactNode => {
+    return action.contents.map((c: Content, contentIdx: number) => {
+      if (
+        (action.destType === destType.SLACK && c.prop === 'channel') ||
+        (action.destType === destType.SLACK && c.prop === 'channel_id')
+      ) {
+        return <React.Fragment key={contentIdx} />
```

Comment:
let's supply the key with some additional info on top of the index, to ensure each key is always unique and can be properly referenced:

```suggestion
        return <React.Fragment key={contentIdx+c.prop} />
```

Reply:
I went back and did this for all service rule files that are using a key :thumbsup:.